documentation/explainers/synthetic_tokens.md

```diff
@@ -0,0 +1,85 @@
+# Overview
+
+Synthetic tokens are collateral-backed tokens whose required backing collateral fluctuates depending on the tokens’ reference index. Synthetic tokens blend features of prediction markets, futures markets, and collateralized loans. 
+
+Some examples of synthetic tokens include: 
+- synthetic real-world assets
+- synthetic cross-chain cryptoassets
+- tracking tokens for various non-tradable indices
+
+Some of the most creative ideas for synthetic tokens fall in the last category. Check out our [Slack channel #tokenideas](https://join.slack.com/t/umaprotocol/shared_invite/zt-7mtxxds5-OIhE~q_WkwGCVNrq0~G~rg) for more project ideas like:
+- tokens that track the future usage of DeFi projects (e.g. assets locked in Uniswap)
+- tokens that track the number of downloads of a Chrome extension (e.g. Metamask)
+- tokens that track the success of trade ideas on r/WallStreetBets 
+
+## Priceless Synthetic Tokens
+“Priceless” synthetic tokens are synthetic tokens that are securely collateralized without an on-chain price feed. These tokens are designed with mechanisms to incentivize token sponsors (those who create synthetic tokens) to properly collateralize their positions. These mechanisms include a liquidation and dispute process that allows token holders to be rewarded for identifying improperly collateralized token sponsor positions. The dispute process relies on an oracle, the UMA DVM, to settle disputes regarding liquidations. 
+
+For more details on how a “priceless” framework can be applied more generally to other types of financial contract templates on the blockchain, check out this blog post. 
+
+## Launching a New Synthetic Token
+
+To launch a new type of synthetic token for which an existing market does not yet exist, that synthetic token’s smart contract must first be parameterized and deployed. Anyone can parameterize and deploy this contract. 
+
+This smart contract can be deployed using UMA’s synthetic token contract template and defining key parameters. Some parameters to highlight are:
+- token’s price identifier (the price feed this token should track)
+- token expiration timestamp
+- token collateralization ratio (e.g. a synthetic token must have collateral worth at least 120% of the price indentifier’s current value) 
+
+This tutorial will show you how to parameterize and deploy the smart contract for a new synthetic token from the command line.
+
+## Creating Synthetic Tokens From an Existing Contract
+After the contract is deployed, anyone can interact with the contract to create synthetic tokens. These actors are called “token sponsors”, since they sponsor the creation of synthetic tokens. [Image]
```

Comment:
```suggestion
After the contract is deployed, anyone can interact with the contract to create synthetic tokens. These actors are called “token sponsors”, since they sponsor the creation of synthetic tokens. 
![Diagram](./synthetic_token_creation.png)
```